.gitignore

```diff
@@ -67,9 +67,20 @@ SwiftScripts/.build-**
 # Local Netlify folder
 .netlify
 
+# Generated js files we don't need committed
+Sources/ApolloCodegenLib/Frontend/dist/
+
 # Local generated packages we don't need in the main project
 Sources/AnimalKingdomAPI/Generated/Package.swift
 Sources/AnimalKingdomAPI/Generated/Package.resolved
-
-# Generated js files we don't need committed
-Sources/ApolloCodegenLib/Frontend/dist/
+Tests/TestCodeGenConfigurations/**/AnimalKingdomAPI/
+Tests/TestCodeGenConfigurations/**/AnimalKingdomAPITestMocks
+Tests/TestCodeGenConfigurations/**/Package.resolved
+Tests/TestCodeGenConfigurations/**/Podfile.lock
+Tests/TestCodeGenConfigurations/**/TestMocks
+Tests/TestCodeGenConfigurations/EmbeddedInTarget-RelativeAbsolute/PackageOne/**/*.graphql.swift
+Tests/TestCodeGenConfigurations/EmbeddedInTarget-RelativeAbsolute/PackageTwo/Sources/PackageTwo
+!Tests/TestCodeGenConfigurations/Other-CustomTarget/AnimalKingdomAPI/AnimalKingdomAPI.h
+!Tests/TestCodeGenConfigurations/Other-CocoaPods/CocoaPodsProject/AnimalKingdomAPI/Schema/SchemaConfiguration.swift
```

Comment:
The Schema configurations are probably always going to need to be kept. If we do end up configuring cache keys for future tests in the other example projects. Should we change this to one line of
```
!Tests/TestCodeGenConfigurations/**/SchemaConfiguration.swift
```

Reply:
Sure, we can commit and track all of the schema configuration files. I only kept the ones that currently had custom logic in them, but yes I agree it's not so obvious to make a change and then wonder why it's not being tracked. Good call!

Reply:
This is done in https://github.com/apollographql/apollo-ios/pull/2991/commits/eaad7d6845a91dc6cec5cc351bf2aeedfd3aceb8.